libs/feature/record/src/lib/state/mdview.facade.ts

```diff
@@ -204,4 +209,17 @@ export class MdViewFacade {
   loadUserFeedbacks(datasetUuid: string) {
     this.store.dispatch(MdViewActions.loadUserFeedbacks({ datasetUuid }))
   }
+
+  /**
+   * FeatureCatalog
+   */
+
+  loadFeatureCatalog(metadataUuid: string, approvedVersion = false) {
+    this.store.dispatch(
+      MdViewActions.loadFeatureCatalogAttributes({
+        metadataUuid,
+        approvedVersion,
+      })
+    )
+  }
```

Comment:
we should not need this on the facade because loading the attributes will not be required by the components; instead, an effect in the state should do it